File: lab7/Recipe.php
<?php


class Recipe implements JsonSerializable
{
    public $id;
    public $name;
    public $author;
    public $type;
    public $description;

    /**
     * Recipe constructor.
     * @param $id
     * @param $name
     * @param $author
     * @param $type
     * @param $description
     */
    public function __construct($id, $name, $type, $author, $description)
    {
        $this->id = $id;
        $this->name = $name;
        $this->type = $type;
        $this->author = $author;
        $this->description = $description;
    }


    public function jsonSerialize()
    {
        return get_object_vars($this);
    }
}

File: lab9/Models/Recipe.cs
namespace lab9.Models
{
    public class Recipe
    {
        public int id;
        public string name;
        public string type;
        public string author;
        public string description;

        public Recipe(int id, string name, string type, string author, string description)
        {
            this.id = id;
            this.name = name;
            this.author = author;
            this.type = type;
            this.description = description;
        }
    }
}